### PR TITLE
FUSETOOLS-2475 - show a marker if the catalog version loaded does not match what is configured inside the pom.xml 

### PR DESCRIPTION
**Pull request by lhein**
see #1237

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/internal/ProjectTemplatesActivator.java [hunk 1]
```diff
@@ -104,4 +146,84 @@ public ProjectTemplatesSharedImages(Bundle pluginBundle) {
...
+				default: // nothing
+			}
+			return false; 
+      	}
+      	
+      	private boolean isRelevantResource(IResource res) {
+      		return 	res != null &&
+      				"pom.xml".equalsIgnoreCase(res.getName()) && 
+      				!"bin".equalsIgnoreCase(res.getParent().getName()) && 
+      				!"target".equalsIgnoreCase(res.getParent().getName());
```
Comment (bfitzpat) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/internal/ProjectTemplatesActivator.java]: Why "bin" and "target" and not just pom.xml? Aren't we just interested in the version changes that start with the pom?

**Comment by bfitzpat** [comment]
Validation for me just keeps going through the same cycle over and over reporting these issues in the console: 
```
Mar 02, 2018 11:05:43 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:05:43 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:05:55 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:05:55 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:06:04 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:06:04 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:06:16 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:06:16 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:06:25 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:06:25 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:06:37 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:06:37 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:07:07 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:07:07 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:07:19 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:07:19 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:07:28 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:07:28 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:07:40 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:07:40 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:07:49 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:07:49 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request
Mar 02, 2018 11:08:01 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: I/O exception (java.net.ConnectException) caught when processing request: Connection timed out: connect
Mar 02, 2018 11:08:01 AM org.apache.commons.httpclient.HttpMethodDirector executeWithRetry
INFO: Retrying request

!ENTRY org.fusesource.ide.camel.model.service.impl 2 0 2018-03-02 11:08:10.586
!MESSAGE Unable to load Camel Catalog for version 2.20.3

!ENTRY org.fusesource.ide.camel.model.service.impl 2 0 2018-03-02 11:08:10.589
!MESSAGE The version 2.20.2 will be used instead.

!ENTRY org.fusesource.ide.camel.model.service.impl 2 0 2018-03-02 11:08:22.152
!MESSAGE Unable to load Camel Catalog for version 2.20.3

!ENTRY org.fusesource.ide.camel.model.service.impl 2 0 2018-03-02 11:08:22.156
!MESSAGE The version 2.20.2 will be used instead.
```

**Comment by bfitzpat** [comment]
Definitely looks like issues with Groovy/Ivy not knowing when to give up the ghost way down in the DefaultCamelCatalog.loadVersion(String) line 163 spot when it's trying to resolve an unreachable camel version. (In my case 2.20.3)

**Comment by apupier** [comment]
closing outdated PR

**Pull request closed without merging**